samples/core/pom.xml

```diff
@@ -145,32 +145,73 @@
     </dependency>
 
     <!-- CXF for REST and Webservices -->
-    <dependency>
-      <groupId>org.apache.cxf</groupId>
-      <artifactId>cxf-rt-frontend-jaxws</artifactId>
-      <exclusions>
-        <exclusion>
-          <groupId>asm</groupId>
-          <artifactId>asm</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.cxf</groupId>
-      <artifactId>cxf-rt-frontend-jaxrs</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.cxf</groupId>
-      <artifactId>cxf-rt-rs-service-description</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.cxf</groupId>
-      <artifactId>cxf-rt-transports-http</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.jaxrs</groupId>
-      <artifactId>jackson-jaxrs-json-provider</artifactId>
-    </dependency>
+
+<dependency>
+
+<groupId>org.apache.cxf</groupId>
+
+<artifactId>cxf-rt-frontend-jaxws</artifactId>
+
+
+<exclusions>
+
+
+<exclusion>
+
+<groupId>asm</groupId>
```

Comment:
Something went wrong with XML formatting here. Please check and avoid empty lines.